experimental/PropertyDDS/packages/property-changeset/src/test/validator/templateValidator.spec.ts

```diff
@@ -1066,10 +1067,12 @@ import {
 						expect(result.isValid).to.equal(false);
 						// console.log(result.errors);
 						expect(result.errors.length).to.equal(5);
-						expect(result.errors[3].message).to.include(
-							"must have required property 'inherits'",
-						);
-						expect(result.errors[4].message).to.include(
+						const error3 = result.errors[3];
+						assert(error3 !== undefined, "error3 is undefined in validate");
```

Comment:
Open question: is this the best pattern to use? (The question is for all code reviewers, not just Rishhi.

Reply:
Not horrible for test code, but not likely what we'd want for production code.
Line 1069 has `expect(result.errors.length).to.equal(5);` if that were `assert(result.errors.length === 5,..);` would `result.errors[3]` be seen as defined? If not, if `errors` was extracted, would the assertion then hold? (I.e. 
```typescript
const { errors } = result;
assert(errors.length === 5, "error count should be 5");
expect(errors[3].message).to.include("must have required property 'inherits'");
```

Reply:
It does not defined if I try `assert(errors.length === 5, "error count should be 5");`

Reply:
Which one would you like me to keep, should I keep both?
```
expect(errors.length).to.equal(5);
assert(errors.length === 5, "error count should be 5");
```